README.md

```diff
@@ -7,6 +7,13 @@ Repository for the DHTech 2022 Hackathon
 
 This software is licensed under the [Apache 2.0 License](LICENSE.md).
 
+## Installation
+
+To install the latest version from GitHub::
```

Comment:
This will install the latest development version, right? So maybe 
```suggestion
To install the latest development version from GitHub::
```

Reply:
yes, that makes sense! (I'm used to my own workflow where we use git flow, so `main` is always the latest release and `develop` is the development version — but I think this wording makes sense here.)

Reply:
We could also move development work to a develop branch, and then only push to main when we have a new versioned release.

Reply:
Just saw @rlskoeser's comment here - I think that flow makes sense here as well

Reply:
ya, we use git flow, too. Makes sense to me!

Reply:
sounds good to me!